.github/workflows/main.yml

```diff
@@ -62,3 +62,52 @@ jobs:
       - run: npm run publish-all
         env:
           NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
+
+  publish-github-container-registry:
+    runs-on: ubuntu-latest
+    if: github.event_name == 'release'
+    environment: release
+    needs: build
+    permissions:
+      contents: read
+      packages: write
+      attestations: write
+      id-token: write
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Log in to the Container registry
+        uses: docker/login-action@v3
+        with:
+          registry: ghcr.io
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
@dsp-ant @jspahrsummers can you tell if there is a GITHUB_TOKEN in secrets, or have @slimslenderslacks help set one up for publishing to the container registry? @aaronpowell has this process publishing on his fork and I can use his image. I think we're close to having this ready to go, but someone with higher magic needs to manage the secrets.

Reply:
That secret is provided by GitHub as part of Actions, it doesn't need to be setup